t/issue/1401.t

```diff
@@ -0,0 +1,86 @@
+
+use Test::More tests => 6;
+
+use strict;
+use warnings;
+use 5.010;
+use autodie qw(:all);
+use English qw($OSNAME -no_match_vars);
+use File::Temp;
+
+use Rex::Commands::Run;
+
+$::QUIET = 1;
+
+my $win = $OSNAME =~ m/^MSWin/mxsi;
+my $quote = q{"};
+my $space = q{ };
+my $empty = q{};
+
+sub command {
+  my ( $path, $exe, @parm ) = @_;
+  return qq{$path/$exe } . join $empty, @parm;
+}
+
+sub relative {
+  return qq{./$_[0]};
```

Comment:
`File::Spec->join()`

Reply:
We might not need the `relative` subroutine, if we `cwd` into the parent of the temp directory first, like this:

```perl
run $command, cwd => dirname($path);
```